nrpy/infrastructures/BHaH/CurviBoundaryConditions/tests/CurviBoundaryConditions_apply_bcs_inner_only__cuda.cu

```diff
@@ -49,9 +49,9 @@ void apply_bcs_inner_only(const commondata_struct *restrict commondata, const pa
   const innerpt_bc_struct *restrict inner_bc_array = bcstruct->inner_bc_array;
   const int num_inner_boundary_points = bc_info->num_inner_boundary_points;
 
-  const size_t threads_in_x_dir = DEFAULT_THREADS_IN_X_DIR;
-  const size_t threads_in_y_dir = DEFAULT_THREADS_IN_Y_DIR;
-  const size_t threads_in_z_dir = DEFAULT_THREADS_IN_Z_DIR;
+  const size_t threads_in_x_dir = BHAH_CURVIBC_INNER_THREADS_IN_X_DIR;
```

Comment:
I like the idea! Just one suggestion: let's make it `THREADS_IN_?_DIR_[suffix]` instead, to make it more quickly identifiable when scanning a function for macros.